practices/java-web/HelloWorldServletMvcWebApplication/src/main/java/com/dowa/java/web/HelloWorldMvcServlet.java

```diff
@@ -20,11 +23,21 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
     @Override
     protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
 
-        RequestDispatcher requestDispatcher = req.getRequestDispatcher("/Hello.jsp");
-
+        RequestDispatcher requestDispatcher = req.getRequestDispatcher("/index.jsp");
         String name = req.getParameter("name");
         req.setAttribute("fakeName", StringUtils.reverse(name));
 
+        StudentRepository repo = new StudentRepository();
+        Student search = null;
+        try {
+            search = repo.findStudentByFirstName(name);
+        } catch (SQLException e) {
+            e.printStackTrace();
```

Comment:
don't eat the exception, do something with it either re-throw it or throw a new exception for example RuntimeException, the objective is to log the real error but to hide it from the user